src/main/java/frc/utils/calibration/Camera/CalibrateCamera.java

```diff
@@ -0,0 +1,139 @@
+package frc.utils.calibration.Camera;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.utils.LimelightHelpers;
+import frc.utils.math.AngleTransform;
+import frc.utils.math.FieldMath;
+import org.littletonrobotics.junction.Logger;
+
+public class CalibrateCamera extends Command {
+
+	private double sumCosX3D = 0, sumSinX3D = 0;
```

Comment:
```suggestion
	private double rollCosSum = 0, rollSinSum = 0;
```
and rename all the other vars in that way, maybe add "cameraPose" somewhere in there but not neccesery